AG-9302 Bullet graph docs

olegat opened the pull request:
https://ag-grid.atlassian.net/browse/AG-9302

- Update _Simple Bullet_ section
- Add _Horizontal Bullet_ section
- Add _Multiple Bullets_ section

olegat committed:
AG-9302 Update 'Simple Bullet' section

olegat committed:
AG-9302 Add 'Multiple Bullets' section

olegat committed:
AG-9302 Add 'Horizontal Bullet' section

alantreadway reviewed (approved):
LGTM, does the multi-chart example work for React/Angular/Vue cases?

olegat commented:
> LGTM, does the multi-chart example work for React/Angular/Vue cases?

Good point. I hadn't checked that. No it doesn't, I'll to find another solution.

olegat committed:
AG-9302 Use chart options to set width/height of bullet examples

This approach it compatible with frameworks (React, Angular, Vue),
althought the example generator will need to be adjusted to center the
chart correctly for these frameworks.

olegat commented:
There's still some issues with React/Angular/Vue, but I'll address these an a follow up PR. I'm going to merge this to collect feedback.
